src/libclient/brushes/brushpresetmodel.cpp

```diff
@@ -18,6 +18,7 @@
 #include <QSqlDatabase>
 #include <QSqlError>
 #include <QSqlQuery>
+#include <QThread>
```

Comment:
iwyu or accident?

Reply:
Leftover attempt of adding retries in addition to error messages, since the error messages made it clear that the problem was that Qt's dirname wasn't working as expected. Removed it.